src/test/java/org/broadinstitute/hellbender/utils/bwa/BwaMemIndexImageCreatorTest.java

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.hellbender.utils.bwa;
+
+import org.broadinstitute.hellbender.BwaMemIntegrationTest;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Test if image can be successfully created.
+ */
+public class BwaMemIndexImageCreatorTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File("src/test/resources/large/");
+    private static final File testReferenceFasta = new File(TEST_DATA_DIR, "human_g1k_v37.20.21.fasta");
+
+    @Override
+    public String getTestedClassName() {
```

Comment:
CommandLineProgramTest has a bit of magic that determines the testedClassName automatically if it's name <NameOfTool>IntegrationTest.  I think this should be renamed still.